pkg/proxy/volume.go

```diff
@@ -336,7 +336,7 @@ func (p *Proxy) RemountReadOnlyVolume(ctx context.Context, req *rpc.RemountVolum
 			"remount,rw",
 			mp.Path,
 		}
-		if _, err := nsexec.Execute("mount", opts, lhtypes.ExecuteDefaultTimeout); err != nil {
+		if _, err := nsexec.Execute([]string{}, "mount", opts, lhtypes.ExecuteDefaultTimeout); err != nil {
```

Comment:
James, you can use nil for the first arg.

Reply:
Modified. Thanks.